### PR TITLE
Some of the changes needed to build on ghc-8.8.1

### PR DESCRIPTION
**Pull request by neurogoo**
Few open questions left:
- text-containers is not updated to ghc-8.8.1 and has not been updated for a while. Maybe just take it out?
- DRBG does not work as it's dependecy cipher-aes128 does not work with cabal 3.0 

Also src/Futurice/Reflection/TypeLits.hs:52:24 gives error that still needs to be looked

### CODE REVIEW

**Comment by neurogoo** [comment]
Also there is no Data.Align.Key anymore so that also needs to be looked on

**Comment by phadej** [comment]
Data.Align.Key was removed, there's `Data.Align.Indexed` in [`semialign-indexed`](https://hackage.haskell.org/package/semialign-indexed) which gives you `ialignWith` (for previous `alignWithKey`).

Should be quite mechanical changes: all the instances should be there. I recommend using `semialign-1.1` and `semialign-indexed-1.1` versions. There was some class renaming, but it all should be in the changelogs.

File: .travis.yml [hunk 1]
```diff
@@ -1,126 +1,177 @@
 # This Travis job script has been generated by a script via
 #
-#   runghc make_travis_yml_2.hs '--config' '--output=.travis.yml' 'cabal.project'
+#   haskell-ci 'cabal.project' '--output' '.travis.yml'
```
Comment (phadej) [inline .travis.yml]: if you rename `cabal.make-travis-yml` to `cabal-haskell-ci` (which is a convention) and use `haskell-ci --config=cabal.haskell-ci cabal.project`, you'll pick options from there. E.g. `branches`.

Comment (neurogoo) [reply .travis.yml]: Thanks, good to know 👍

**Pull request merged**